fakerseedsintro.md

```diff
@@ -0,0 +1,15 @@
+Populating seeds with faker users
```

Comment:
Can we rename this file into a TODO.md instead? It's a much more common practice, and it can serve as a point where people pick off tasks.